.github/workflows/config.yml

```diff
@@ -80,7 +80,7 @@ jobs:
 
       - name: Run buildstockbatch
         run: |
-          pip install git+https://github.com/NREL/buildstockbatch.git@develop
+          pip install git+https://github.com/NREL/buildstockbatch.git@total_energy_use
```

Comment:
Temporary change for testing the appropriate bsb.

Reply:
Yep, change this back to `develop` after https://github.com/NREL/buildstockbatch/pull/280 is merged.